src/api/operations/client.fragments.graphql

```diff
@@ -80,6 +80,12 @@ fragment ClientNameDobVet on Client {
   veteranStatus
 }
 
+fragment ClientNameDobSsn on Client {
```

Comment:
Interesting, we don't usually take this approach of defining a fragment that is not used in any queries. I think we should either (1) define this type in the code, ` type ClientNameDobSsn = ClientNameFragment & {ssn?: string, dob?:string}` or, (2)  add `... ClientNameDobSsn` fragment to the AllEnrollmentDetails.client fields